flink-core/src/main/java/org/apache/flink/core/fs/EntropyInjector.java

```diff
@@ -89,6 +89,10 @@ public static Path removeEntropyMarkerIfPresent(FileSystem fs, Path path) {
 
 	// ------------------------------------------------------------------------
 
+	public static boolean isEntropyInjecting(FileSystem fs) {
+		return getEntropyFs(fs) == null;
```

Comment:
`getEntropyFs(fs) != null;`

Reply:
You are right, definitely. This hints that I need to add a test.